### ISSUE TITLE
showSelectedFirst new prop on Select component

### ISSUE THREAD

**Issue by lukzerom** [ant-design/ant-design#33171]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
When selecting many items from Select dropdown  - some of them may not fit inside input (as tags) and user is not sure which items she/he choose. The only solution is to search for item or  scroll throughout all dropdown and find one . In case od many items in dropdown list, this can be hard. This prop will show all selected items on the top od dropdown after close / open. The state od top selected items will change with every new dropdown open.

### What does the proposed API look like?
showSelectedFirst = boolean

**Comment by afc163** [comment]
It is hard to understand, could you provide a example picture for your need?

**Comment by lukzerom** [comment]
> It is hard to understand, could you provide a example picture for your need?

Sure, screen below 😌. I suggest new boolean prop to show all selected items in top of items dropdown  - like below. You can "uncheck" items, and after dropdown close / open they will sort again -> all selected items to top
<img width="534" alt="Zrzut ekranu 2021-12-6 o 16 39 58" src="https://user-images.githubusercontent.com/55348672/144876252-303e1658-5f54-444a-8821-078896fcff46.png">
.

**Comment by afc163** [comment]
Is this demo you want? https://ant.design/components/select/#components-select-demo-multiple

**Comment by lukzerom** [comment]
> Is this demo you want? https://ant.design/components/select/#components-select-demo-multiple

Yes, but with selected items always on the top od the dropdown - so as not to look for them in the whole dropdown.
